### PR TITLE
Update user guide for Genome Assembly Assessment

### PR DESCRIPTION
**Pull request by AnnaSyme**
Workflow has been updated; updating user-guide to match.

### CODE REVIEW

File: genome_assembly/assembly_qc.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ Once you have checked the genome assembly, you may decide that:
 
 ### How to cite the workflow
 
-> Price, G., & Syme, A. (2024). Genome-assessment-post-assembly. WorkflowHub. <a href="https://doi.org/10.48546/WORKFLOWHUB.WORKFLOW.403.4" ga-product="workflow_doi" ga-id="403_assembly_QC">https://doi.org/10.48546/WORKFLOWHUB.WORKFLOW.403.4</a>
+> Kate Farquharson, Johan Gustafsson, Gareth Price, Simon Tang, Anna Syme (2024). Genome-assessment-post-assembly. WorkflowHub. <a href="https://doi.org/10.48546/WORKFLOWHUB.WORKFLOW.403.7" ga-product="workflow_doi" ga-id="403_assembly_QC">https://doi.org/10.48546/WORKFLOWHUB.WORKFLOW.403.4</a>
```
Comment (supernord) [inline genome_assembly/assembly_qc.md:31]:
```suggestion
> Kate Farquharson, Gareth Price, Simon Tang, Anna Syme (2024). Genome-assessment-post-assembly. WorkflowHub. <a href="https://doi.org/10.48546/WORKFLOWHUB.WORKFLOW.403.7" ga-product="workflow_doi" ga-id="403_assembly_QC">https://doi.org/10.48546/WORKFLOWHUB.WORKFLOW.403.4</a>
```

Comment (supernord) [reply genome_assembly/assembly_qc.md:31]: Also, checking the WorkflowHub entry: Simon's name doesn't appear in the list of creators.
Does this need to be updated?

Comment (AnnaSyme) [reply genome_assembly/assembly_qc.md:31]: Yep I have updated it in WorkflowHub now.

Comment (AnnaSyme) [reply genome_assembly/assembly_qc.md:31]: Do you not want to be listed?

Comment (supernord) [reply genome_assembly/assembly_qc.md:31]: I don't think I've contributed enough to be listed as a creator for the workflow.

Comment (AnnaSyme) [reply genome_assembly/assembly_qc.md:31]: Ok no problemo

**Pull request merged**